tests/test_tiatoolbox.py

```diff
@@ -350,7 +439,7 @@ def test_command_line_slide_info(_response_all_wsis):
 
 
 def test_command_line_read_region(_response_ndpi, tmp_path):
-    """Test the Read Region CLI."""
+    """pytest OpenSlide read_region CLI."""
```

Comment:
Not sure about this verbing of pytest

Reply:
I will change it to Test and make it consistent